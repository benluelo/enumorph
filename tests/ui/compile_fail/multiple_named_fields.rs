#[derive(enumorph::Enumorph)]
enum Enum {
    A { b: u8, c: u8 },
}

fn main() {}
