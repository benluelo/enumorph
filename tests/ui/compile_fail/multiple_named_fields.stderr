error: only variants with one field are supported
 --> tests/ui/compile_fail/multiple_named_fields.rs:3:7
  |
3 |     A { b: u8, c: u8 },
  |       ^^^^^^^^^^^^^^^^
